packages/three-vrm-springbone/src/utils/lowestCommonAncestor.ts

```diff
@@ -0,0 +1,25 @@
+import type * as THREE from 'three';
+
+/**
+ * Finds the lowest common ancestors of the given objects, if it exists.
+ * @param objects The objects to find the lowest common ancestor for.
+ */
+export function lowestCommonAncestor(objects: THREE.Object3D[]): THREE.Object3D | null {
+  const sharedAncestors = new Map<THREE.Object3D, number>();
+  for (const object of objects) {
+    let current: THREE.Object3D | null = object;
+    do {
+      if (!sharedAncestors.has(current)) {
+        sharedAncestors.set(current, 0);
+      }
+
+      const newValue = sharedAncestors.get(current)! + 1;
```

Comment:
```suggestion
      const newValue = (sharedAncestors.get(current) ?? 0) + 1;
```

does this do the same job?